ui/lib/core/addon/components/masked-input.hbs

```diff
@@ -32,7 +32,7 @@
       aria-label={{or @name "masked input"}}
       {{on "change" this.onChange}}
       {{on "keyup" (fn this.handleKeyUp @name)}}
-      data-test-textarea={{or @name ""}}
+      data-test-input={{or @name ""}}
```

Comment:
There was a lot of logic in tests because this selector differed from the standard pattern, updated it and relevant selectors to make future tests easier to write

Reply:
I've also run into this. Thank you!